plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityDisabledSlots.java

```diff
@@ -85,91 +101,109 @@ public static void registerTags() {
         // @group properties
         // @description
         // If the entity is an armor stand, returns a list of its disabled slots in the form slot/action|...
+        // Consider instead using <@link tag EntityTag.disabled_slots_data>.
         // -->
         PropertyParser.<EntityDisabledSlots, ObjectTag>registerTag(ObjectTag.class, "disabled_slots", (attribute, object) -> {
-
-            // <--[tag]
-            // @attribute <EntityTag.disabled_slots.raw>
-            // @returns ElementTag(Number)
-            // @mechanism EntityTag.disabled_slots_raw
-            // @group properties
-            // @description
-            // If the entity is an armor stand, returns its raw disabled slots value.
-            // See <@link url https://minecraft.fandom.com/wiki/Armor_Stand/ED>
-            // -->

```

Comment:
Don't delete deprecated meta - replace it, search for `// @deprecated` to find examples of deprecated tag meta